modules/virtual_environments/auto-venv/venv_helpers.nu

```diff
@@ -23,6 +23,20 @@ export def venv-is-active [] {
     '__auto_venv' in (overlay list)
 }
 
+export def 'venv-create' [] {
+# Creates a virtual environment under the current directory
+	let venv_path = (pwd)
+	let venv_name = ((pwd) | path split | last) 
```

Comment:
you can also use `pwd | str trim | path basename` here :+1:

Reply:
Shouldn't that be `$env.PWD`?
The `pwd` command is Unix specific.

Reply:
> Shouldn't that be `$env.PWD`? The `pwd` command is Unix specific.
>
lol i somehow forgot about `$env.PWD` in the comment above :laughing: 

it should be
```
$env.PWD | path basename
```
:wink: 

in fact, `$env.PWD == (pwd | str trim)` appears to be always true :ok_hand:

Reply:
Thanks! Done.

Reply:
:+1: